pegasus/sites.v3/code.org/public/custom-certificates.haml

```diff
@@ -1,5 +1,6 @@
 ---
 Title: Print Certificates
+post: true
```

Comment:
Could we call the header allow_post rather than just post? This will make the meaning more obvious.